README.md

```diff
@@ -19,6 +19,27 @@ You may need to install/update .NET Framework for this.
 
 After opening the project successfully, you should be able to run the game and play my fun little cat RPG identical to the Java version.
 
+### For Mac Developers
```

Comment:
Can you make a subsection for windows developers like you did here for Mac, just to make it more clear to someone trying to run it that there are OS specific instructions? Wouldn't want someone with a Mac to find the setup instructions and follow line by line only to realize it won't work and then later hit the Mac instructions...I've seen it happen so many times while teaching lol.